build/git_windows.go

```diff
@@ -0,0 +1,26 @@
+package build
```

Comment:
Not sure if we should put this in this package or util one. Maybe on buildkit as I think it could be useful for Windows support (cc @TBBle @gabriel-samfira)